framework/core/src/Foundation/Console/InfoCommand.php

```diff
@@ -168,4 +186,27 @@ private function identifyDatabaseVersion(): string
     {
         return $this->db->getPdo()->getAttribute(PDO::ATTR_SERVER_VERSION);
     }
+
+    /**
+     * Some extensions/packages might be overriding the session.handler binding.
+     * So we have to check if the driver used is configured or hardcoded.
+     */
+    private function identifySessionDriver(): string
+    {
+        $defaultDriver = $this->session->getDefaultDriver();
+        $driver = Arr::get($this->config, 'session.driver', $defaultDriver);
+        $this->session->driver($driver);
+        $configuredDriver = isset($this->session->getDrivers()[$driver]) ? $driver : $defaultDriver;
+
+        // Get class name
+        $handlerName = get_class($this->sessionHandler);
+        // Drop the namespace
+        $handlerName = Str::afterLast($handlerName, '\\');
+        // Lowercase the class name
+        $handlerName = strtolower($handlerName);
+        // Drop everything like queue SyncQueue, RedisQueue
+        $handlerName = str_replace('sessionhandler', '', $handlerName);
+
+        return $configuredDriver !== $handlerName ? "$handlerName <comment>(code override, configured to <options=bold,underscore>$configuredDriver</>)</comment>" : $configuredDriver;
```

Comment:
Let's use an if(configuredDriver === handlerName) { return configuredDriver} return ...;

That should improve readability a bit/

Reply:
should be much much more readable now.